readthedocs/projects/static-src/projects/css/_content.less

```diff
@@ -213,3 +213,79 @@ select {
     padding-left: 0;
   }
 }
+
+.projects-filter {
+    &:before {
+        font-family: FontAwesome;
+        content: "\f07b";
+        padding-right: 10px;
```

Comment:
Una cosetta di dettaglio, anche per @protoroto, anche se stiamo ancora impostando questo layout temporaneo che non richiede troppi perfezionismi: di norma usiamo una _baseline_ di 8px e quindi multipli di esso (quindi in linea di massima tutti i 10px diventeranno 8px, 15px > 16px, 20px > 24px, e così via).

Oltre a questo, Bootstrap Italia [mette a disposizione classi per le spaziature](https://italia.github.io/bootstrap-italia/docs/utilities/spaziature/), per cui qui ad esempio avreste potuto usare `pr-2` (che setta il padding a destra di 8px).